tests/unit/components/HdLazyImage.spec.js

```diff
@@ -0,0 +1,25 @@
+import { mount } from '@vue/test-utils';
+import HdLazyImage from '@/components/HdLazyImage.vue';
+import ITEMS from '@/stories/mocks/GALLERY_ITEMS';
+
+describe('HdLazyImage', () => {
+  it('renders with the image as background', () => {
+    const wrapper = mount(HdLazyImage, {
+      src: ITEMS[0].image,
+      background: true,
+    });
+
+    expect(wrapper.html()).toMatchSnapshot();
+  });
+
+  it('renders with the image as picture element, sources and source set', () => {
+    const wrapper = mount(HdLazyImage, {
+      src: ITEMS[3].image,
```

Comment:
Same here, props have to be passed under `propsData`

Reply:
Done!